### ISSUE TITLE
[Question]: Yes! In case there's still install issues after that let me know.

### ISSUE THREAD

**Issue by soundsnow** [Rakashazi/emu-ex-plus-alpha#1129]
### Describe your question

App not installed as package conflicts Recent nightly builds
The signing certificate was updated from 1024-bit to 2048-bit automatically by the Android dev tools so it looks like you guys will have to remove the current apps and install the APK fresh to avoid security errors.

**Comment by TortugaV6SC** [comment]
The problem I'm having is similar. "app not installed" error even when previous app removed. So I can't even install the old version to uninstall it. I think part of the issue lies with "keep old files?" Question. I didn't delete the old files and tried to install the updated APK.

**Comment by TortugaV6SC** [comment]
Is there a way to uninstall the app signatures manually on a phone that isn't rooted? Or does anyone have the ngpemu 1.5.67 APK? It says expired https://github.com/Rakashazi/emu-ex-plus-alpha/actions/runs/3350753054

**Comment by beefmechado** [comment]
> The problem I'm having is similar. "app not installed" error even when previous app removed. So I can't even install the old version to uninstall it. I think part of the issue lies with "keep old files?" Question. I didn't delete the old files and tried to install the updated APK. EDIT: I don't have the old version of the APK to uninstall completely (neoGeo pocket 1.567) because I deleted it and can't redownload it only the new version. PS... Sorry for sloppy explanation. Need coffee. I'll revise later

Having the same problem, although mine was for gbc. I thought I'd save my settings, etc, but now I can't even install the app.

**Comment by TortugaV6SC** [comment]
> > The problem I'm having is similar. "app not installed" error even when previous app removed. So I can't even install the old version to uninstall it. I think part of the issue lies with "keep old files?" Question. I didn't delete the old files and tried to install the updated APK. EDIT: I don't have the old version of the APK to uninstall completely (neoGeo pocket 1.567) because I deleted it and can't redownload it only the new version. PS... Sorry for sloppy explanation. Need coffee. I'll revise later
> 
> Having the same problem, although mine was for gbc. I thought I'd save my settings, etc, but now I can't even install the app.

I fixed the issue. It's the APK signature. So essentially I had to find an APK with a matching signature and it should install then uninstall BUT completely do NOT leave data.

Then you should be able to install the new APK with a different signature

**Comment by beefmechado** [comment]
> > > The problem I'm having is similar. "app not installed" error even when previous app removed. So I can't even install the old version to uninstall it. I think part of the issue lies with "keep old files?" Question. I didn't delete the old files and tried to install the updated APK. EDIT: I don't have the old version of the APK to uninstall completely (neoGeo pocket 1.567) because I deleted it and can't redownload it only the new version. PS... Sorry for sloppy explanation. Need coffee. I'll revise later
> > 
> > 
> > Having the same problem, although mine was for gbc. I thought I'd save my settings, etc, but now I can't even install the app.
> 
> I fixed the issue. It's the APK signature. So essentially I had to find an APK with a matching signature and it should install then uninstall BUT completely do NOT leave data.
> 
> Then you should be able to install the new APK with a different signature

Where you able to find the APK with the matching signature? All the nightlies seem to get deleted here after a new one comes out.

**Comment by TortugaV6SC** [comment]
> > > > The problem I'm having is similar. "app not installed" error even when previous app removed. So I can't even install the old version to uninstall it. I think part of the issue lies with "keep old files?" Question. I didn't delete the old files and tried to install the updated APK. EDIT: I don't have the old version of the APK to uninstall completely (neoGeo pocket 1.567) because I deleted it and can't redownload it only the new version. PS... Sorry for sloppy explanation. Need coffee. I'll revise later
> > > 
> > > 
> > > Having the same problem, although mine was for gbc. I thought I'd save my settings, etc, but now I can't even install the app.
> > 
> > 
> > I fixed the issue. It's the APK signature. So essentially I had to find an APK with a matching signature and it should install then uninstall BUT completely do NOT leave data.
> > Then you should be able to install the new APK with a different signature
> 
> Where you able to find the APK with the matching signature? All the nightlies seem to get deleted here after a new one comes out.

Yeah, so sometimes the newer APKs MIGHT have the same signature on the nightlies. Other than that I'm unsure, sorry. Maybe the developer can help.
